CHANGELOG.md

```diff
@@ -1,9 +1,42 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+This project adheres to [Semantic Versioning](http://semver.org/)
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
-## 1.2.0 2017-05-13
+## [Unreleased]
+
+## [2.0.0] - 2018-02-07
+### Breaking Change
+- Split up the variables used to determine if a host gets rabbitmq/redis for more flexibility in deployments. (@tculp)
+  `sensu_deploy_rabbitmq` and `sensu_deploy_redis` are now `sensu_deploy_rabbitmq_server` and `sensu_deploy_redis_server` respectively.
+  See the [role variable documentation](https://github.com/sensu/sensu-ansible/blob/master/docs/role_variables.md) for details on the parameters.
+- Redis on Ubuntu will now be configured to bind to `0.0.0.0` to ensure accessiblity and to match the other supported OS configurations (@tculp)
+- Updated the supported Ansible version to the last two stable releases (currently that's Ansible 2.3 and 2.4) (@jaredledvina)
+  Please note that we have not explicitly broken support for running this role on versions of Ansible <2.3. However, we will only
+  be activly supporting the last two stable Ansible releases to reduce the maintenance burden.
```

Comment:
why is this on a newline?

Reply:
Was making it easier to view in Vim and I was thinking Markdown might play nice and leave it there but, I see in the rendered view that's not the case. So, nuke 'em and put it all on one line. I think the rendering looks alright.

Reply:
ya when dealing with unordered lists in markdown it just is easier to put each item on its own line and let it wrap. Markdown solved that problem a long time ago.